File: SoloLearn/Skee-Ball/SkeeBall.c
#include <stdio.h>

int main() {
    int score, tickets, cost;
    scanf("%d", &score);
    tickets = score / 12;
    scanf("%d", &cost);
    if (tickets < cost)
        printf("Try again");
    else
        printf("Buy it!");
    return 0;
} 

File: SoloLearn/Skee-Ball/SkeeBall.java
import java.util.Scanner;

public class SkeeBall {

    public static void main(String[] args) {
        Scanner input = new Scannner(System.in);
        int score = input.nextInt(), tickets = score / 12, cost;
        input.nextLine();
        cost = input.nextInt();
        if (tickets < cost)
            System.out.println("try again");
        else    
            System.out.println("Buy it!");
    }
}